### PR TITLE
Documentation Fixes

### PR DESCRIPTION
**Pull request by piraka9011**
Addresses #36.

## Changes

* Update README with better install instructions 
* Tell users that sysroot path needs to be absolute

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,14 +18,16 @@ The following instructions have been tested on Ubuntu Xenial (16.04) and Bionic
 # Install cross compilation toolchain
 sudo apt-get update
 sudo apt-get install -y build-essential cmake git wget curl lsb-core bash-completion \
-    qemu-user-static g++-aarch64-linux-gnu g++-arm-linux-gnueabihf python3-pip htop
+    qemu-user-static python3-pip htop g++-arm-linux-gnueabihf 
+# If you are on a non-arm system
+sudo apt-get install g++-aarch64-linux-gnu
```
Comment (zmichaels11) [inline README.md]: no `-y` here?

**Comment by piraka9011** [comment]
> There was also mention of needing to copy the dockerfile into sysroot/ directory that is not documented here. Do we need to add that in as well?

No, the script should handle this automatically [here](https://github.com/aws-ros-dev/cross_compile/blob/eef947c195ebfc77f7dafb19f0cf235c0c63582f/cross_compile/sysroot_compiler.py#L239).

Not sure why that issue happened.

**Comment by prajakta-gokhale** [comment]
> No, the script should handle this automatically [here](https://github.com/aws-ros-dev/cross_compile/blob/eef947c195ebfc77f7dafb19f0cf235c0c63582f/cross_compile/sysroot_compiler.py#L239).
> 
> Not sure why that issue happened.

Yeah, that's what I thought. If it worked on your local testing in a new workspace, it was probably a user workspace related issue.

**Comment by deitry** [comment]
> the script should handle this automatically here.

Maybe the reason was linked to relative paths that I tried to pass as `--sysroot-path`, so it can't handle it.
I added another PR that fixes dependancy on absolute paths.
https://github.com/ros-tooling/cross_compile/pull/39

**Comment by thomas-moulard** [comment]
@piraka9011 You should have write access to this repo - do you mind pushing this on this repo to get the CI passing and close this PR? I think it's an OK fix until we get to fix the CI script.

**Comment by piraka9011** [comment]
Closing in favor of #37

**Pull request closed without merging**